OpenIPSL/Electrical/Controls/PSSE/TG/BaseClasses/GGOV1/Turbine.mo

```diff
@@ -153,13 +161,12 @@ Turbine dynamic system for the general purpose Turbine/Governor models GGOV1 and
 </blockquote>
 <h5>Linearisation</h5>
 <p>
-If one wants to use this model in combination with linear analysis then the replacable <code>delay</code> block
-needs to get changed to <a href=\"modelica://OpenIPSL.Electrical.Controls.PSSE.TG.BaseClasses.GGOV1.DelayModelChoices.PadeDelay\">OpenIPSL.Electrical.Controls.PSSE.TG.BaseClasses.GGOV1.DelayModelChoices.PadeDelay</a>
-block instead (only allowed matching replacement).
+If one wants to use this model in combination with linear analysis then the parameter <code>delayType</code> 
+needs to get changed to \"Pade Delay\".
 
 The Pade Delay model herein has as default n=2 and m=n.
 
-Linearisation will fail if <code>Teng = 0</code>. The default time-delay is set to <code>Teng = C.eps</code>. 
+Linearisation will fail if <code>delayTime = 0</code>. The default time-delay is set to <code>delayTime = C.eps</code>. 
```

Comment:
You can get back to `Teng` here, since it is the correct name for the parameter.

Reply:
Actually, it needs to get changed to `padeDelay.delayTime` to be correct. If we leave `Teng`, then the users would have to perform two actions, first change the delay type **and** change `Teng`.  As said before, if PadeDelay is chosen, Teng should be replaced by `C.eps` by default.
I've updated the documentation now.

Reply:
Yeah, same here. Got it now, thanks.